atlite/data.py

```diff
@@ -135,9 +129,6 @@ def cutout_prepare(cutout, features=slice(None), tmpdir=None, overwrite=False):
 
             ds.to_netcdf(cutout.path, mode=mode)
 
-    if not keep_tmpdir:
-        rmtree(tmpdir)
-
```

Comment:
As we have an automated deletion of the tmp files in `era5.py` and all other modules (`sarah`, `gebco`) don't produce any tempfiles, I removed the tmpdir creation and deletion. If tmpdir is None, the datasets modules should delete any produced tmp files automatically.  If tmpdir is specified, the files will be kept.

Reply:
Not a good idea! Have been down that road before.

> A direct consequence for making the cutout preparation work for sarah was to let go of the cleanup tmp files automatically once the file handle is released (the weakref.finalizer(noisy_unlink, ...) bit), since dask is often configured to use multiple processes and then whether a file handle is released in one process is not a good indicator for whether we still need the tmp file. Thus, I was forced to switch to a create all files in a temporary directory and clean up the directory when the cutout is ready scheme.
from https://github.com/PyPSA/atlite/issues/47#issuecomment-532579112